src/select/select.tsx

```diff
@@ -1247,47 +1250,61 @@ export default class Select<T = unknown> extends Component<SelectProps<T>, Selec
     switch (this.props.type) {
       case Type.INPUT_WITHOUT_CONTROLS:
       case Type.INPUT: return (
-        <div
-          ref={this.nodeRef}
-          className={classNames(classes, styles.inputMode)}
-          data-test={dataTests('ring-select', dataTest)}
-        >
-          {shortcutsEnabled && (
-            <Shortcuts
-              map={this.getShortcutsMap()}
-              scope={this.shortcutsScope}
+        <>
+          <div
+            ref={this.nodeRef}
+            className={classNames(classes, styles.inputMode)}
+            data-test={dataTests('ring-select', dataTest)}
+          >
+            {shortcutsEnabled && (
+              <Shortcuts
+                map={this.getShortcutsMap()}
+                scope={this.shortcutsScope}
+              />
+            )}
+            <Input
+              {...ariaProps}
+              height={this.props.height}
+              autoComplete="off"
+              id={this.props.id}
+              onClick={this._clickHandler}
+              inputRef={this.filterRef}
+              disabled={this.props.disabled}
+              value={this.state.filterValue}
+              borderless={this.props.type === Type.INPUT_WITHOUT_CONTROLS}
+              style={style}
+              size={Size.FULL}
+              onChange={this._filterChangeHandler}
+              onFocus={this._focusHandler}
+              onBlur={this._blurHandler}
+
+              // Input with error style without description
+              error={this.props.error != null ? '' : null}
+              label={this.props.type === Type.INPUT ? this._getLabel() : null}
+              placeholder={this.props.inputPlaceholder}
+              onKeyDown={this.props.onKeyDown}
+              data-test="ring-select__focus"
+              enableShortcuts={shortcutsEnabled
+                ? Object.keys({
+                  ...this.getShortcutsMap(),
+                  ...this._popup?.list?.shortcutsMap
+                })
+                : undefined}
+              afterInput={this.props.type === Type.INPUT && iconsNode}
             />
+            {this._renderPopup()}
+          </div>
+          {this.props.error && (
```

Comment:
I used the same method as added error for this component
https://github.com/JetBrains/ring-ui/blob/c40ca5009296e983b7491f729daed01268460ba3/src/editable-heading/editable-heading.tsx#L168-L170